core/src/main/scala/neotypes/Driver.scala

```diff
@@ -8,10 +8,37 @@ import org.neo4j.driver.v1.{AccessMode, Driver => NDriver}
 import scala.jdk.CollectionConverters._
 import scala.language.higherKinds
 
+/** A neotypes driver for accessing the neo4j graph database
+  * A driver wrapped in the resource type can be created using the neotypes GraphDatabase
+  * {{{
+  *    val driver = GraphDatabase[F]("bolt://localhost:7687").driver
+  * }}}
+  *
+  * @param driver neo4j driver
+  * @tparam F effect type for driver
+  * @note the resource type for scala.Future has no failure support, see <a href = https://neotypes.github.io/neotypes/docs/alternative_effects.html>alternative effects</a>
```

Comment:
I would rather write something like this:
_"When your effect type is `scala.Future` there is no concept of **Resource**. For more information see ...."_